README.md

```diff
@@ -67,9 +67,16 @@ At this point you can run the Gibbon executable:
 
     $ cabal v2-exec -w ghc-9.0.1 gibbon -- -h
 
-If you'd like to run the testsuite, you can do so with:
+And add gibbon to PATH
+```
+$ export PATH = "{ ... }/gibbon/dist-newstyle/build/{architecture}/ghc-{version}/gibbon-0.2/x/gibbon/build/gibbon/gibbon:$PATH"
+```
 
-    $ cd $GIBBONDIR && ./run_all_tests.sh
+If you'd like to run the testsuite, you can do so with:
```

Comment:
Did you run into a problem with the `run_all_tests` script? I'll admit that I haven't ran it in a while, but the goal is to keep it up-to-date.